labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,22 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val string1: String = "ejp mysljylc kd kxveddknmc re jsicpdrysi"
+    val string1Decoded: String = "our language is impossible to understand"
+    val string2: String = "rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd"
+    val string2Decoded: String = "there are twenty six factorial possibilities"
+    val string3: String = "de kr kd eoya kw aej tysr re ujdr lkgc jv"
+    val string3Decoded: String = "so it is okay if you want to just give up"
+    val list1 = string1.zip(string1Decoded)
+    val list2 = string2.zip(string2Decoded)
+    val list3 = string3.zip(string3Decoded)
+    val list = list1.union(list2).union(list3)
```

Comment:
could put them all into one long string, but this works too and is a bit more readable since you don't get a super long line of code